azure-pipelines.yml

```diff
@@ -26,8 +26,11 @@ jobs:
       Python39:
         python.version: '3.9'
         NOXSESSION: 'tests-3.9'
+#      Python310:
+#        python.version: '3.10.0b2'
+#        NOXSESSION: 'tests-3.10'
     # Increase the maxParallel value to simultaneously run the job for all versions in the matrix (max 10 for free open-source)
-    maxParallel: 4
+    maxParallel: 10
```

Comment:
If I recall, last I checked the maximum you get for free is 4, but I setup our account when I worked at Microsoft, so who knows if we are exempt from that limit.

Reply:
The comment above says up to 10. Is that wrong then?

Reply:
The comment is probably correct.  I was probably remembering an earlier limitation.